docs/l2-l3.rst

```diff
@@ -92,7 +92,7 @@ Create Default Gateway
 
 4. Click :guilabel:`Submit`.
 
-The default gateway is created on the BIG-IP device and is added to the application database. For an HA setup, the default gateway will automatically sync to the BIG-IP peer device.
+The default gateway is created on the BIG-IP device and is added to the application database. For an HA setup, the default gateway will automatically sync to the BIG-IP peer device. F5 ACI ServiceCenter has added support for brownfield default gateway from v2.9 where default gateway will automatically sync to FASC database once user navigates to L2-L3 tab. FASC has added support to allow any Default Gateway name to be synced to FASC database (Until version 2.8, the only default gateway name supported for  was **apic-default-gateway**). 
```

Comment:
F5 ACI ServiceCenter v2.9 added support for a brownfield default gateway, where the default gateway automatically syncs to the FASC database once the user navigates to the L2-L3 tab. FASC added support to allow any Default Gateway name to be synced to the FASC database (in previous versions, the only supported default gateway name was **apic-default-gateway**).